cmd/byohctl/service/constants.go

```diff
@@ -13,7 +13,7 @@ const (
 	DefaultFilePerms = 0644
 
 	// ByohAgentDebPackageURL is the URL to download the agent package
-	ByohAgentDebPackageURL = "quay.io/platform9/byoh-agent-deb:0.1.200"
+	ByohAgentDebPackageURL = "quay.io/platform9/byoh-agent-deb:0.1.276"
```

Comment:
Can we read this tag from a config file / parameter so that we can update it as metadata without changing code?